paper.tex

```diff
@@ -307,41 +330,65 @@ \subsection*{Unambiguous low-level representation}
+ewing2010msms,zhou2018popdemog}. However, these methods
+currently have little traction as they are all either directly coupled
+to an internal simulation method or to the specific syntax
+of a given simulator. Another approach that has been taken to
+simplify running such simulations is to provide an API in a
+scripting language such as Python or R, providing a flexible
+interface either to a built-in simulation
+engine~\citep{thornton2014cpp,kelleher2016efficient,haller2017flexible}
+% jk: are/were there other simulation frontends out there?
+or as a frontend for other simulators~\citep{staab2016coala}.
+While APIs have many advantages and have grown in popularity in
+recent years, the demographic model descriptions are tied
+to the details of both the underlying simulation engines
+and the programming language in question.
+
+Stable and healthy software ecosystems require standard interchange
+formats, allowing for the development of high-quality and long-lasting
+tools that produce and consume the standard.
+Demographic models are a key part of population genetics research,
+and to date the transfer of inferred models to downstream simulations
+has been ad-hoc, and conversions between the many different ways
+of expressing such models is both labour intensive and fraught with errors.
+The proposed Demes standard is an attempt to bridge this gap
+between inference and simulation, and also to provide the foundations
+for a sustainable ecosystem of tools built around this data model.
+Table~\ref{tab-software} shows some initial infrastructure that we have
+built as part of developing Demes, but many other useful tools
+can be envisaged that either produce or consume this format.
+
+Reproducibility is a significant problem throughout the
+sciences~\citep{baker20161} and various measures have been
+proposed to increase the likelihood of researchers being
+able to replicate results in the
+literature~\citep{munafo2017manifesto}. The most basic requirement
+for reproducibility is that we must be able to state precisely what
+the result in question is. The lack of standardisation in how
+complex demographic models are communicated today, and the lack of
+precision in the published model descriptions means that it is difficult
+to replicate analyses, or reproduce those models for later simulation.
+Thus, we hope that the Demes standard introduced here will be widely adopted
+by inference methods and the results reported in publications
+routinely included as supplemental material or uploaded to a data
+repository in Demes format.
+
+\jkcomment{sketch of last paragraph}
+Another basic requirement for scientific progress in the inference
+of demographic models is that we should be able to compare the output
+of one method to the output of another, or to compare an inferred
+demography to some simulated ground truth. Demes facilitates this,
+paving the way for things like methods competitions. PopSim
```

Comment:
Another benefit of the ease of comparison between methods, for developers, is that new simulation methods under development can more easily be compared to existing methods. (in my own experience, I very often would compare methods I've worked on in dadi or moments to msprime or fwdpy11.. previously if they disagreed, I wouldn't be completely sure if it was an implementation issue or an underlying issue with my simulator).

Essentially, can highlight those comparison benefits for both end-product users and developers.